packages/i18n/src/index.js

```diff
@@ -17,7 +17,10 @@ import sprintfjs from 'sprintf-js';
  */
 const DEFAULT_LOCALE_DATA = {
 	'': {
-		plural_forms: ( n ) => ( n === 1 ? 0 : 1 ),
+		/** @param {number} n */
+		plural_forms( n ) {
```

Comment:
This fixes a local implicity any error, but the package still requires `noImplicitAny` because `tannin` and `memize` are untyped.

Reply:
> This fixes a local implicity any error, but the package still requires `noImplicitAny` because `tannin` and `memize` are untyped.

Sounds like a nice weekend project for me 😄 

(Previous WIP: https://github.com/aduth/tannin/tree/add/typescript-declarations)

Reply:
While I suppose [you'd already seen it](https://github.com/aduth/tannin/pull/4), just for posterity's sake, both Tannin and Memize now provide type definitions:

https://github.com/aduth/tannin/commit/b5f5fd958abae0d092baa6f867c660d98561bf2f
https://github.com/aduth/memize/commit/3a6f1cd38e25cf37716d2398c6ab098992acccfb

Reply:
Yes, that's amazing! Thanks for adding those. We'd need `tannin@1.2.0` and `memize@1.1.0` it looks like. I won't take on upgrading those packages in Gutenberg myself right now, but when they land `i18n` should be able to drop the `noImplicityAny`.

Reply:
Circled back to this: https://github.com/WordPress/gutenberg/pull/21224